packages/compass-e2e-tests/index.ts

```diff
@@ -18,9 +22,13 @@ import {
 } from './helpers/compass';
 import ResultLogger from './helpers/result-logger';
 
+// https://github.com/nodejs/node/issues/40537
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
We probably shouldn't be introducing new usage of this hack and instead address the actual issue? We only accepted that we'd need to add it in a few places in order to get things working quickly, not as a real fix

Reply:
How do you fix the real issue?

Reply:
> How do you fix the real issue? 

Depends on what we're trying to do/where it's failing – if we want to connect to a MongoDB server that's listening on `127.0.0.1` only, we should connect to it using `127.0.0.1`, not `localhost`.

> The ideal there (assuming node's own dns module isn't going to do it) would be for the driver to switch out to a dns module that does, right?

No, for connections made using the Node.js driver, it just needs to implement the spec and connect to all addresses returned from `getaddrinfo`; https://jira.mongodb.org/browse/NODE-5754 tracks that (the title is a bit misleading in the sense that this isn't about IPv4 vs IPv6 specifically; the "I want the driver to attempt to use all A and AAAA records for each hostname" bit is the relevant one).